blockchain/chain.py

```diff
@@ -26,10 +26,14 @@ class Block:
     """
         Create a new block in chain with metadata
     """
-    def __init__(self, data, index=0):
+    def __init__(self, uid_epita, email_epita, nom, prenom, image, index=0):
```

Comment:
I think this commit wasn't supposed to be in this PR, it's only relevant for EPITA students :)